### ISSUE TITLE
Missing InterlockedCompareExchange8 definition

### ISSUE THREAD

**Issue by jkotas** [microsoft/WindowsAppSDK#4074]
### Describe the bug

InterlockedCompareExchange8 definition is missing in Windows SDK.

All other InterlockedCompareExchangeXXX and InterlockedExchangeXXX variants are present. InterlockedCompareExchange8 omission seems to be a bug.

### Steps to reproduce the bug

1. Copy and paste the following code into Visual Studio C++ console project:
```
#include <windows.h>

static CHAR a;

int main()
{
    InterlockedCompareExchange8(&a, 1, 0);
}
```
2. Compile

### Expected behavior

Expected behavior: Project compiles succesfully
 
Actual behavior: Project fails to compile due to missing InterlockedCompareExchange8 definiton.

### NuGet package version

None

### Windows version

Windows 11 version 22H2 (22621, 2022 Update)

### IDE

Visual Studio 2022

**Comment by DarranRowe** [comment]
Are you are aware of the intrinsic definition of _InterlockedCompareExchange8 in intrin.h?
By the looks of it, the intrinsics in winnt.h basically use the same definitions as in intrin.h. This means that you can define it yourself.

**Comment by jkotas** [comment]
Yes, it is what we have done as a workaround: https://github.com/dotnet/runtime/pull/92974/files#diff-16c3a3017bc0de55f505efda656370c3bc632e1c9404b025c45dda36d374b4a1R260-R263
